setup.py

```diff
@@ -14,69 +14,71 @@
 
 from setuptools import setup, find_packages
 
-_current_jaxlib_version = '0.3.14'
+_current_jaxlib_version = "0.3.14"
 # The following should be updated with each new jaxlib release.
-_latest_jaxlib_version_on_pypi = '0.3.14'
-_available_cuda_versions = ['11']
-_default_cuda_version = '11'
-_available_cudnn_versions = ['82', '805']
-_default_cudnn_version = '82'
-_libtpu_version = '0.1.dev20220627'
+_latest_jaxlib_version_on_pypi = "0.3.14"
+_available_cuda_versions = ["11"]
+_default_cuda_version = "11"
+_available_cudnn_versions = ["82", "805"]
+_default_cudnn_version = "82"
+_libtpu_version = "0.1.dev20220627"
 
 _dct = {}
-with open('jax/version.py') as f:
-  exec(f.read(), _dct)
-__version__ = _dct['__version__']
-_minimum_jaxlib_version = _dct['_minimum_jaxlib_version']
+with open("jax/version.py") as f:
+    exec(f.read(), _dct)
+__version__ = _dct["__version__"]
+_minimum_jaxlib_version = _dct["_minimum_jaxlib_version"]
 
 setup(
-    name='jax',
+    name="jax",
     version=__version__,
-    description='Differentiate, compile, and transform Numpy code.',
-    author='JAX team',
-    author_email='jax-dev@google.com',
+    description="Differentiate, compile, and transform Numpy code.",
+    author="JAX team",
+    author_email="jax-dev@google.com",
     packages=find_packages(exclude=["examples"]),
-    package_data={'jax': ['py.typed']},
-    python_requires='>=3.7',
+    package_data={"jax": ["py.typed"]},
+    python_requires=">=3.7",
     install_requires=[
-        'absl-py',
-        'numpy>=1.19',
-        'opt_einsum',
-        'scipy>=1.5',
-        'typing_extensions',
-        'etils[epath]'
+        "absl-py>=1.1.0",
```

Comment:
Is this version of absl-py actually needed? (It's probably not a bad idea to set a minimum version, though, since we will most likely accidentally depend on absl-py features at some point in the latest version.)